sdk/include/reactos/subsys/sm/smmsg.h

```diff
@@ -1,14 +1,17 @@
 /*
- * PROJECT:         ReactOS Windows-Compatible Session Manager
- * LICENSE:         BSD 2-Clause License
- * FILE:            include/reactos/subsys/sm/smmsg.h
- * PURPOSE:         SMSS (SB and SM) Message Format
- * PROGRAMMERS:     Alex Ionescu
+ * PROJECT:     ReactOS Windows-Compatible Session Manager
+ * LICENSE:     BSD 2-Clause License (https://spdx.org/licenses/BSD-2-Clause.html)
+ * PURPOSE:     SMSS Client (SB and SM) Message Format
+ * COPYRIGHT:   Copyright 2012-2013 Alex Ionescu <alex.ionescu@reactos.org>
+ *              Copyright 2021 Hervé Poussineau <hpoussin@reactos.org>
+ *              Copyright 2022 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
  */
-#pragma once
+
 #ifndef _SM_MSG_
 #define _SM_MSG_
 
+#pragma once
```

Comment:
What is the benefit of using pragma once **and** include-guards at the same time within the same file? Looks odd to me.
Is there any reason why we can't just use pragma once everywhere? (my personal favorite)
Is there any reason why we can't just use include guards everywhere? (2nd best)